frontend/inbox/src/pages/Contacts/index.tsx

```diff
@@ -61,13 +62,15 @@ const Contacts = (props: ContactsProps) => {
   }, [currentTableData]);
 
   useEffect(() => {
-    lastPage % fetchNextPage == 0 && listContacts();
+    lastPage % fetchNextPage == 0 &&
```

Comment:
nitpick: It would be great to save `lastPage % fetchNextPage == 0` in a variable so that it easy to understand quickly what this statement means